e2e/app/page/authenticated-page.ts

```diff
@@ -57,5 +59,12 @@ export default abstract class AuthenticatedPage extends BasePage {
     return getPropValue<string>(username, 'innerText');
   }
 
+  async closeHelpSidebarIfOpen(): Promise<void> {
```

Comment:
Help Tip sidebar opens automatically when user logins first time. An open sidebar could "confuse" function to use the wrong sidebar when the runtime sidebar is the target.
So if it is found to be open, close it first.